### PR TITLE
Player 4887

### PR DESCRIPTION
**Pull request by nzaitcev**
* [PLAYER-4888](https://jira.corp.ooyala.com/browse/PLAYER-4888) Enable/disable CC via JavaScript API
* [PLAYER-4887](https://jira.corp.ooyala.com/browse/PLAYER-4887) Enabling setting Closed Captions when casting

additional changes for [this PR](https://git.corp.ooyala.com/projects/PBW/repos/mjolnir/pull-requests/620/overview)

includes reverted changes from 4888

### CODE REVIEW

File: js/controller.js [hunk 1]
```diff
@@ -2694,6 +2695,10 @@ module.exports = function(OO, _, $, W) {
       }
     },
 
+    toggleClosedCaptions: function() {
+      this.mb.publish(OO.EVENTS.TOGGLE_CLOSED_CAPTIONS);
+    },
```
Comment (zelibobla) [inline js/controller.js]: Guess this should be covered by tests?

Comment (nzaitcev) [reply js/controller.js]: done)

File: tests/html5skin-test.js [hunk 2]
```diff
@@ -204,6 +205,12 @@ describe('Controller', function() {
       expect(controller.state.closedCaptionOptions.enabled).toBe(true);
       expect(controller.state.persistentSettings.closedCaptionOptions.enabled).toBe(true);
     });
+
+    it('should publish TOGGLE_CLOSED_CAPTIONS event on toggleClosedCaptions', () => {
+      controller.toggleClosedCaptions();
+      expect(OO.mb.publish).toHaveBeenCalledTimes(2);
```
Comment (ooyala-amohnatkin) [inline tests/html5skin-test.js]: why 2 times?

**Comment by zelibobla** [comment]
Thank you for tests. It's important.

**Pull request merged**